mempool/src/logging.rs

```diff
@@ -1,14 +1,17 @@
 // Copyright (c) The Diem Core Contributors
 // SPDX-License-Identifier: Apache-2.0
 
-use crate::shared_mempool::{peer_manager::BatchId, types::ConsensusRequest};
+use std::{fmt, time::SystemTime};
+
 use anyhow::Error;
-use diem_config::{config::PeerNetworkId, network_id::NetworkId};
+use serde::Serialize;
+
+use diem_config::network_id::{NetworkId, PeerNetworkId};
 use diem_logger::Schema;
 use diem_types::{account_address::AccountAddress, on_chain_config::OnChainConfigPayload};
 use mempool_notifications::MempoolCommitNotification;
-use serde::Serialize;
-use std::{fmt, time::SystemTime};
+
```

Comment:
nit: why the spacing? is this some new formatting thing? I thought we put all imports next to each other

Reply:
Oh, must have been CLion, I used it to move my `PeerNetworkId` code and it did a lot of weird things